### PR TITLE
Add respectivity async handling patterns by languages.

### PR DESCRIPTION
**Pull request by kekyo**
# Async in depth summarise focusing only C#

Async in depth summarise focusing only C# language, so add "for C#" and another languages have respective pattern.

# Details

If we are using for F#, async pattern is "async workflow" and using Async type better than Task type. But I feel this article suit for C#, one of the representative example.

I changed:

* Write to it's C# pattern.
* Write no explicitly  languages, but it's difference.

### CODE REVIEW

File: docs/standard/async-in-depth.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ ms.assetid: 1e38f9d9-8f84-46ee-a15f-199aec4f2e34
 
 # Async in depth
 
-Writing I/O- and CPU-bound asynchronous code is straightforward using the .NET Task-based async model. The model is exposed by the `Task` and `Task<T>` types and the `async` and `await` language keywords. This article explains how to use .NET async and provides insight into the async framework used under the covers.
+Writing I/O- and CPU-bound asynchronous code is straightforward using the .NET Task-based async model. The model is exposed by the `Task` and `Task<T>` types and the `async` and `await` C# language keywords. (Any other languages have respective pattern.) This article explains how to use .NET async and provides insight into the async framework used under the covers.
```
Comment (rpetrusha) [inline docs/standard/async-in-depth.md]: better: "`async` and `await` C# language keywords." --> "`async` and `await` keywords in C# and Visual Basic."
"Any other languages have respective pattern." --> (Other languages may have a different syntax or different keywords.)

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for suggesting this change, @keyko! `async` and `await` are also keywords in Visual Basic, so it's not C# only. Could you revise your PR by incorporating the changes that I've suggested?

**Comment by GuardRex** [comment]
Extending @kekyo's point a little bit: Cross-references to the language-specific docs, such as:

* Asynchronous programming - https://docs.microsoft.com/en-us/dotnet/csharp/async
* Async Programming in F# - https://docs.microsoft.com/en-us/dotnet/fsharp/tutorials/asynchronous-and-concurrent-programming/async
* Asynchronous Programming with Async and Await (Visual Basic) - https://docs.microsoft.com/en-us/dotnet/visual-basic/programming-guide/concepts/async/

... seem missing here in a *See also* section at the bottom. If adding them makes sense, then the line @kekyo added ...

> (Any other languages have respective pattern.)

... could call out the language-specific resources in that section (and we'd possibly even remove the "C#" that @kekyo added in this PR, since that's somewhat obvious) ...

> (Language-specific resources are found in the [See also](#see-also) section.)

(:point_up: something along those lines)

```
## See also

[Asynchronous programming in C#](https://docs.microsoft.com/en-us/dotnet/csharp/async)   
[Async Programming in F#](https://docs.microsoft.com/en-us/dotnet/fsharp/tutorials/asynchronous-and-concurrent-programming/async)   
[Asynchronous Programming with Async and Await (Visual Basic)](https://docs.microsoft.com/en-us/dotnet/visual-basic/programming-guide/concepts/async/)
```

Last question: If that's a good add here, wouldn't it also be a good add these over in [Asynchronous Programming Patterns](https://docs.microsoft.com/en-us/dotnet/standard/asynchronous-programming-patterns/) with a little better cross-linking among the three docs on async in the .NET Platform Guide section?

cc/ @cartermp @BillWagner @rpetrusha

**Comment by GuardRex** [comment]
@kekyo If you want to add a *See also* section here (and at the bottom of [Asynchronous Programming Patterns](https://docs.microsoft.com/en-us/dotnet/standard/asynchronous-programming-patterns/)), there's an important thing to note about lists without bullet points. When you list links back-to-back without a blank line between them, such as ...

```
## See also

[Asynchronous programming in C#](https://docs.microsoft.com/en-us/dotnet/csharp/async)   
[Async Programming in F#](https://docs.microsoft.com/en-us/dotnet/fsharp/tutorials/asynchronous-and-concurrent-programming/async)   
[Asynchronous Programming with Async and Await (Visual Basic)](https://docs.microsoft.com/en-us/dotnet/visual-basic/programming-guide/concepts/async/)
```
... you must include three spaces at the end of lines before the last line. This is currently undocumented (but we'll document it soon). So for those three links, set them up like this ...

```
## See also

[TITLE_HERE](<LINK_HERE>)<PUT_THREE_SPACES_HERE>
[TITLE_HERE](<LINK_HERE>)<PUT_THREE_SPACES_HERE>
[TITLE_HERE](<LINK_HERE>)
```

**Comment by rpetrusha** [comment]
@GuardRex, I think your suggested improvements to the topic are valuable. Could you open an issue that addresses them? @kekyo, thank you so much for pointing out the single-language focus of this topic and for making corrections. We'll merge your PR. Thanks again for your contribution!

**Review by rpetrusha — APPROVED** [review]

**Comment by kekyo** [comment]
I respond late sorry, thanks merged my proposal!

**Pull request merged**